doc/source/examples/01_getting_started/05_plotter_picker.mystnb

```diff
@@ -179,6 +179,30 @@ plotter = PlotterHelper(use_trame=True)
 plotter.plot(plot_list)
 ```
 
+## Clipping objects
+
+You can clip any object represented in the plotter by defining a ``Plane`` object that
+intersects the target object.
+
+```{code-cell} ipython
+
+from ansys.geometry.core.math import Plane, Point3D
+ph = PlotterHelper()
+
+# Define pyansys-geometry box
```

Comment:
```suggestion
# Define pyansys-geometry box
```
Is pyansys-geometry needed? If so, can it be PyAnsys Geometry or should in instead be tagged as a code entity given that ``pyansys-geometry`` (the package name) is used as a code entity.